README.md

```diff
@@ -186,4 +180,4 @@ This project is [MIT](./LICENSE) licensed.
 
 _NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [ ] I would kindly suggest you work on the MIT license link of your project as currently, the link is broken and its throwing error 404.

Error 👇 

![mit](https://user-images.githubusercontent.com/78634841/222820882-b483dd5e-92ac-40c2-bee2-d10bd16252d2.png)